pkg/adaptation/plugin.go

```diff
@@ -249,33 +290,35 @@ func (p *plugin) connect(conn stdnet.Conn) (retErr error) {
 }
 
 // Start Runtime service, wait for plugin to register, then configure it.
```

Comment:
no start/stop for wasm only configure maybe update the function definition and add a todo if that will be needed